.github/workflows/price-feeder-docker.yml

```diff
@@ -38,7 +37,7 @@ jobs:
       - name: Build and push
         uses: docker/build-push-action@v3
         with:
-          context: ./contrib/images/price-feeder.Dockerfile
+          context: ./contrib/images/price-feeder.dockerfile
```

Comment:
Can you check once , I think it should be `contrib/images/price-feeder.dockerfile`